addons/calendar/models/calendar_event.py

```diff
@@ -393,14 +396,27 @@ def _set_discuss_videocall_location(self):
         This is done by design to prevent users not being able to join a discuss meeting because the base event of the recurrency was deleted.
         """
         if not self.access_token:
-            self.access_token = uuid.uuid4().hex
+            self.access_token = self._default_access_token()
         self.videocall_location = f"{self.get_base_url()}/{self.DISCUSS_ROUTE}/{self.access_token}"
 
     @api.model
     def get_discuss_videocall_location(self):
         access_token = uuid.uuid4().hex
         return f"{self.get_base_url()}/{self.DISCUSS_ROUTE}/{access_token}"
 
+    def _init_column(self, column_name):
+        """ Initialize the value of the given column for existing rows.
+            Overridden here because we skip generating unique access tokens
+            for potentially tons of existing event, should they be needed,
+            they will be generated on the fly.
+        """
+        if column_name != 'access_token':
+            super(Meeting, self)._init_column(column_name)
+
+    def _generate_access_token(self):
+        for event in self:
+            event.access_token = self._default_access_token()
+
```

Comment:
_generate_access_token is only called by the appointment controller. I am not sure it is necessary to move it to the calendar module.